File: UI/Web/src/app/manga-reader/_models/layout-mode.ts
/**
 * How to layout pages for reading
 */
export enum LayoutMode {
    /**
     * Renders a single page on the renderer. Cover images will follow splitting logic.
     */
    Single = 1,
    /**
     * Renders 2 pages side by side on the renderer. Cover images will not split and take up both panes.
     */
    Double = 2,
    /**
     * Renders 2 pages side by side on the renderer. Cover images will not split and take up both panes. This version reverses the order and is used for Manga only
     */
    DoubleReversed = 3,
    /**
     * Renders 2 pages side by side on the renderer. Cover images will split.
     */
    DoubleNoCover = 4,

}

File: Kavita.Models/Entities/Enums/LayoutMode.cs
using System.ComponentModel;

namespace Kavita.Models.Entities.Enums;

public enum LayoutMode
{
    [Description("Single")]
    Single = 1,
    [Description("Double")]
    Double = 2,
    [Description("Double (manga)")]
    DoubleReversed = 3
}
